app/gui2/src/components/GraphEditor/GraphNodes.vue

```diff
@@ -48,7 +48,6 @@ const uploadingFiles = computed<[FileName, File][]>(() => {
     :node="node"
     :edited="id === graphStore.editedNodeInfo?.id"
     :graphNodeSelections="props.graphNodeSelections"
-    :data-node="id"
```

Comment:
It is very helpful in debugging. We could use `:data-testid="node(${id})"` if you want to have something standardized.

Reply:
This is a duplicate; the `GraphNode` component defines `data-node-id`.